### PR TITLE
Create issue templates

### LINKED ISSUE #11: Create issue templates
## Description 
Currently there are no issue templates for this repo.

## Steps to reproduce
Click **New Issue** button on GitHub repo.

## Current result
No issue templates are used.

## Desired result
The user should be able to choose from several issues types (e.g., **Bug Report**, **Feature Request**, **Help & Other Questions**)

## Affected version
* [x] ?.?.?

### PR DESCRIPTION
**Pull request by david-poindexter**
Resolves #11

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+## Affected version
+<!-- Check all that apply and add more if necessary -->
+
+* [x] 9.2.1
+* [x] 9.2
+* [x] 9.1.1
+* [ ] 9.1
+* [ ] 9.0
+
```
Comment (ohine) [inline .github/ISSUE_TEMPLATE/feature-request.md]: Please add v9.2.2 or remove this section completely. I don't see much value honestly in this for a new feature. Right?

File: .github/ISSUE_TEMPLATE/help-and-other-questions.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+## Affected version
+<!-- Check all that apply and add more if necessary -->
+
+* [x] 9.2.1
+* [x] 9.2
+* [x] 9.1.1
+* [ ] 9.1
+* [ ] 9.0
+
```
Comment (ohine) [inline .github/ISSUE_TEMPLATE/help-and-other-questions.md]: Please add v9.2.2

**Comment by nvisionative** [comment]
@mitchelsellers and @ohine my apologies for the slow response.  I am just now sorting through over 1000 GitHub notifications from DNNSoftware related repos!  😆  I agree with the request to just remove the "Affected Version" and "Browser" sections.  It really doesn't make sense for EVS.  I'll get this updated shortly.  Thanks!

**Comment by hismightiness** [comment]
What is this waiting on now?

**Comment by nvisionative** [comment]
@mitchelsellers and @ohine - the requested changes were made 21 days ago.  Is there something else you need on this one?

**Comment by mitchelsellers** [comment]
@nvisionative This is waiting for the implementation of the CLA agreement, however, we know that you have signed. 

@ohine Can you approve so I can get the second approval here?

**Comment by nvisionative** [comment]
@mitchelsellers - ah, I don't see a prompt for CLA on my end, but happy to sign again.

**Comment by mitchelsellers** [comment]
@nvisionative That's the problem!  It isn't enabled yet!

**Comment by david-poindexter** [comment]
@mitchelsellers any chance of this getting another approval and merged?

**Pull request merged**